DSL/Ruuter/POST/services/status.yml

```diff
@@ -227,7 +227,17 @@ rename_intent_to_tmp:
       current_path: "/mock1/data/nlu/${service_name + '_nlu' + old_file_end}"
       new_path: "/mock1/data/nlu/removed/${service_name}_nlu.tmp"
   result: rename_intent_result
-  next: return_ok
+  next: change_training_status
+
+change_training_status:
+  call: http.post
+  args:
+    url: "[#SERVICE_RUUTER]/training-status"
+    body:
+      serviceId: ${id}
+      status: true
```

Comment:
Is it intentional that this always sets the `status` to `true`, or was it overlooked?

Reply:
Yes, because whenever you change the service status, you modify bot files, so the bot needs to be retrained so in this case it should be true in any status change

Reply:
Got it, it makes sense now.

Reply:
Got it, it makes sense.